src/main/java/org/spongepowered/api/data/manipulator/mutable/item/MapItemData.java

```diff
@@ -26,13 +26,56 @@
 
 import org.spongepowered.api.data.manipulator.DataManipulator;
 import org.spongepowered.api.data.manipulator.immutable.item.ImmutableMapItemData;
+import org.spongepowered.api.data.value.mutable.ListValue;
+import org.spongepowered.api.data.value.mutable.Value;
 import org.spongepowered.api.item.ItemTypes;
 import org.spongepowered.api.item.inventory.ItemStack;
 
 /**
- * Represents an {@link DataManipulator} hosting the specific map
- * information of an {@link ItemStack} of the type {@link ItemTypes#FILLED_MAP}.
- * // TODO for modwizcode
+ * Represents an {@link DataManipulator} hosting the specific map information of
+ * an {@link ItemStack} of the type {@link ItemTypes#FILLED_MAP}.
  */
 public interface MapItemData extends DataManipulator<MapItemData, ImmutableMapItemData> {
+
+    /**
+     * Gets this map's {@link Integer} x center.
+     *
+     * @return The x center of the map.
+     */
+    Value<Integer> xCenter();
+
+    /**
+     * Gets this map's {@link Integer} z center.
+     *
+     * @return The z center of the map.
+     */
+    Value<Integer> zCenter();
+
+    /**
+     * Gets this map's {@link Integer} dimension.
+     *
+     * @return The dimension ID of the map.
+     */
+    Value<Integer> dimensionId();
+
+    /**
+     * Gets this map's {@link Integer} scale.
+     *
+     * @return The scale of the map.
+     */
+    Value<Integer> scale();
+
+    /**
+     * Gets the visible players by name {@link String} on this map.
+     *
+     * @return The list of players visible.
+     */
+    ListValue<String> visiblePlayers();
+    
+    /**
+     * Gets the colors {@link Byte} on this map.
```

Comment:
This is quite literally the worst way to phrase "here's how to make a map! admire my list of byte values!"